adapters/faraday/lib/opentelemetry/adapters/faraday.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Adapters
+    module Faraday
+      module_function
+
+      TRACER_NAME = 'faraday'
+
+      def install(config = {name: TRACER_NAME,
+                            version: tracer_version})
```

Comment:
Do the `name` and `version` need to be overridable by consumers of the adapter?

Reply:
@fbogsany this was removed after our SIG work session.